src/coreclr/tools/aot/ILCompiler.Build.Tasks/ILCompiler.Build.Tasks.csproj

```diff
@@ -8,30 +8,24 @@
     <OutputPath>$(RuntimeBinDir)/ilc-published/netstandard</OutputPath>
     <Configurations>Debug;Release;Checked</Configurations>
     <PlatformTarget>AnyCPU</PlatformTarget>
-
-    <!-- Workaround for https://github.com/dotnet/corert/issues/6306 -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <SkipSigning>true</SkipSigning>
```

Comment:
```suggestion
```
Since you have deleted the link tracking the workaround. This workaround should not be needed anymore now that we are in dotnet/runtime.